engine/utils/sourcemaps/prelude.ml

```diff
@@ -0,0 +1,4 @@
+include Base
+
+let ( << ) f g x = f (g x)
+let ( >> ) f g x = g (f x)
```

Comment:
Is this not already in utils?

Reply:
Yeah, but `sourcemaps` doesn't depend (and cannot depened) on hax-engine :(